package.json

```diff
@@ -20,9 +25,22 @@
     "plugins": {
       "release-it-lerna-changelog": {
         "infile": "CHANGELOG.md",
-        "launchEditor": false
+        "launchEditor": true
       },
-      "release-it-yarn-workspaces": true
+      "release-it-yarn-workspaces": {
+        "workspaces": [
+          "addon"
+        ],
+        "additionalManifests": {
+          "versionUpdates": [
+            "package.json"
+          ],
+          "dependencyUpdates": [
+            "package.json",
+            "test-app/*/package.json"
```

Comment:
@AnastasiiaPlutalova I think this should be `"test-app/package.json"` as there is no another nested folder